src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/Hosted-CSharp.sln

```diff
@@ -2,11 +2,11 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.0.0
 MinimumVisualStudioVersion = 16.0.0.0
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorHosted-CSharp.Server", "BlazorHosted-CSharp.Server\BlazorHosted-CSharp.Server.csproj", "{650B3CE7-2E93-4CC4-9F46-466686815EAA}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorHosted-CSharp.Server", "Server\BlazorHosted-CSharp.Server.csproj", "{650B3CE7-2E93-4CC4-9F46-466686815EAA}"
```

Comment:
If this csproj only exists conditionally then this project definition should also be conditional.

Reply:
Question, do we need a solution with 1 project? The goal here is to have either a generated solution with 1 or 3 depending on "host" option

Reply:
So in the template, I leave out the sln, so it won't be created if hosted = false